### PR TITLE
Adding Database or Dbms related parameters just for show.

### PR DESCRIPTION
**Pull request by npetzall**
* Parameters are added thru a delegate and are not used currently.
* They will be visible in usage report.

Not all but a start.

### CODE REVIEW

**Comment by npetzall** [comment]
Relates to issue #255

**Comment by npetzall** [comment]
There's a lot of commented code,  since jcommander isn't used yet all the possible names can't be used. Since most of the configuration is fetched from Configuration.java

Also there are new configuration like -connProp that i would like to add.

**Comment by npetzall** [comment]
```
$ java -jar target/schemaspy-6.0.0-rc2.jar -help           
  ____       _                          ____
 / ___|  ___| |__   ___ _ __ ___   __ _/ ___| _ __  _   _
 \___ \ / __| '_ \ / _ \ '_ ` _ \ / _` \___ \| '_ \| | | |
  ___) | (__| | | |  __/ | | | | | (_| |___) | |_) | |_| |
 |____/ \___|_| |_|\___|_| |_| |_|\__,_|____/| .__/ \__, |
                                             |_|    |___/

                                              6.0.0-rc2

SchemaSpy generates an HTML representation of a database schema's relationships.

INFO  - Starting Main v6.0.0-rc2 on XXX with PID 3871 (XXX/schemaspy/target/schemaspy-6.0.0-rc2.jar started by XXX in XXX/schemaspy)
INFO  - The following profiles are active: default
INFO  - Usage: java -jar target/schemaspy-6.0.0-rc2.jar [options]
  Options:
    ?, -?, /?, -h, help, -help, --help
      show general usage and available options
    -dbHelp, -dbhelp, --dbHelp, --dbhelp
      Show built-in database types and their required connection parameters
    -host, schamspy.host
      Dbms host
    -cat, schemaspy.cat
      Catalog used for narrowing schema
    -X, schemaspy.columnExclusions
      Regular expression for columns to exclude from all relationship diagrams.
      Default: [^.]
    -connprops, schemaspy.connprops
      Path to properties file containing properties for making the jdbc connection
      Default: {}
    -db, schemaspy.db
      Name of database to connect to
    -dbThreads, schemaspy.dbThreads
      Number of threads to use when querying dbms (think connections)
    -debug, --debug, debug, schemaspy.debug
      Enable debug logging
      Default: false
    -dp, schemaspy.dp
      Override jdbc driver path (dbms java client)
    -x, schemaspy.indirectColumnExclusions
      Regular expression for columns to exclude from relationship diagrams where the specified columns aren't directly 
      referenced by the focal table.
      Default: [^.]
    -meta, schemaspy.meta
      Path to schema-meta xml or folder containing [schema].meta.xml files
    -norows, schemaspy.norows
      Skip fetching number of rows in tables
      Default: false
    -noviews, schemaspy.noviews
      Skip views when querying database
      Default: false
    -o, --outputDirectory, outputDirectory, schemaspy.o, schemaspy.outputDirectory
      directory to place the generated output in
    -p, schemaspy.p
      Dbms credentials
    -port, schemaspy.port
      Dbms listening port
    -s, schemaspy.s
      Name of the schema to analyze (defaults to the specified user)
    -sso, --single-sign-on, schemaspy.sso, schemaspy.single-sign-on
      Remove requirement for user
      Default: false
    -t, schemaspy.t
      DatabaseType(Dbms Configuration) to use
    -I, schemaspy.tableExclusions
      Regular expression for tables to exclude
      Default: <empty string>
    -i, schemaspy.tableInclusions
      Regular expression for tables to include
      Default: .*
    -u, schemaspy.u
      Dbms credentials

Go to http://schemaspy.org for a complete list/description of additional parameters.

Sample usage using the default database type (implied -t ora):

 java -jar schemaSpy.jar -db mydb -s myschema -u devuser -p password -o output
Some databaseTypes requires additional arguments they are supplied using '-[key] [value]'
INFO  - Started Main in 1.872 seconds (JVM running for 2.772)
```

File: src/main/java/org/schemaspy/cli/DbmsCommandLineArguments.java [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+                    "-t", //"--database-type", "database-type",
+                    "schemaspy.t"//, "schemaspy.database-type"
+            },
+            descriptionKey = "databaseType"
+    )
+    private String databaseType;
+    @Parameter(
+            names = {
+                    "-host", //"--host", "host",
+                    "schamspy.host"
```
Comment (danwatt) [inline src/main/java/org/schemaspy/cli/DbmsCommandLineArguments.java]: Misspelling

Comment (npetzall) [reply src/main/java/org/schemaspy/cli/DbmsCommandLineArguments.java]: Horrible misspelling, good catch, thanks for having a look.

**Comment by npetzall** [comment]
Closing this, will do proper after version 6 has been release. We might want to remove some options, also we can introduce the additional aliases.

**Pull request closed without merging**